noaOrMlnx/sonic-buildimage
# Change update_feature_state call to pass False as default if feature has no 'has_timer' field

**- Why I did it**
In order to make hostcfgd not to fail if feature has no 'has_timer' field
**- How I did it**
Change the update_feature_state call to pass False by default.
**- How to verify it**
Enable feature that has no 'has timer' field and make sure hostcfgd not failing. 
**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
